### DISCUSSION TITLE
When I click on  the bottom “Show PK-Analysis”

### DISCUSSION THREAD

**Discussion by LiuYX1998** [Open-Systems-Pharmacology/Forum#1147] [Questions & Problems]
![image](https://user-images.githubusercontent.com/97145041/155491900-9a457aac-905d-4cf2-a943-c219af8bf86a.png)
![image](https://user-images.githubusercontent.com/97145041/155491931-168683c2-f7d0-4253-bb83-1c1cd97e33cd.png)
![image](https://user-images.githubusercontent.com/97145041/155493555-9f496354-3c87-4ec1-b974-94eceda22952.png)

When I click on  the bottom “Show PK-Analysis” ，it shows that i can't run and analyze. 
What should I do to solve the problem?
Kind regards

**Comment by RobinM92** [comment]
When incorporating the observed data, try to map it to your simulated molecule (which should included the molecular weight), otherwise PKSim does not know what the MW is of your observations and is not able to transform your observed concentrations from ng/ml to µM

**Reply by Anhng1234** [reply]
Could you show me the way to map the observed data to my simulated molecule? I have the same problem above. Thank you so much for your help. I am looking forward to hear from you

**Reply by RobinM92** [reply]
Of course. Right-click on your observed data in the observed data building block (BB) and click edit. Then on the left hand side of the panel you will see the meta-data associated with the data. There you can click 'add meta data' and you type 'Molecule' in the name field and select your compound in the value field. 
Next time you implement data it would be better to immediately link it to the correct simulated quantities to avoid this. Also, if you have multiple datasets you can right-click on the observed data building block itself and select 'edit all meta data'. 
Good luck!
